blockstack/lib/config.py

```diff
@@ -284,8 +289,128 @@ def get_genesis_block_stages():
 def get_all_genesis_block_patches():
     return GENESIS_BLOCK_PATCHES
 
+def load_genesis_bulk_address_lines(path, line_start, line_end):
+    """
+    Read a list of lines from a file, from the given starting line (inclusive) to the given ending line (exclusive).
+    The file should be a sequence of newline-separated b58check addresses.
+    Aborts on failure.
+    """
+    if not os.path.exists(path):
+        print >> sys.stderr, 'FATAL: no such file or directory: "{}"'.format(path)
+        os.abort()
+
+    f = open(path, 'r')
```

Comment:
Python best practice is to use the `with` keyword, instead of managing closes manually. No need to change if we're not using that pattern anywhere else.

Reply:
Fixed